scenes/game/local_game.gd

```diff
@@ -1,3 +1,11 @@
+# This is the main game engine, responsible for both
+# game mechanics and generating AI actions. 
```

Comment:
This doesn't generate AI actions at all.
The UI (game.gd) calls AIPlayer to generate the AI actions and passes them through GameWrapper as if they were any other player.